### PR TITLE
Added text effects

### PR DESCRIPTION
**Pull request by elegantShock2258**
Added code for the text effects: 

1) Framed
2) Encircled
3) Overlined

### CODE REVIEW

File: src/main/java/com/diogonunes/jcolor/Attribute.java [hunk 1]
```diff
@@ -30,6 +30,7 @@ public static Attribute BOLD() {
         return new SimpleAttribute("1");
     }
 
+
```
Comment (elegantShock2258) [inline src/main/java/com/diogonunes/jcolor/Attribute.java]: looks like this happened accidentally

**Comment by dialex** [comment]
I'll have a look during the weekend. I can't merge it without updating the tests 😇

**Comment by dialex** [comment]
Good news: this is merged!
Bad news: I can't release the new version due to [a Maven Central bug](https://issues.sonatype.org/browse/OSSRH-66257?focusedCommentId=1098027&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-1098027) :(

I'm not sure how to proceed, as that's out of my control and it's waiting for a solution for more than 6 months. I'll try to find an alternative, but that might take weeks. Just to manage your expectations.

**Comment by dialex** [comment]
Good news: found a workaround and [5.2.0](https://github.com/dialex/JColor/releases/tag/v5.2.0) was released!
It takes a while to appear in maven central. 

Thanks for your contribution 😉

**Pull request merged**